README.md

```diff
@@ -78,6 +78,8 @@ version = "3.16"
 dependencies = [
   "glib-sys/v3_16"
 ]
+# Add features to the "dox" one:
+extra_dox_features = ["whatever"]
```

Comment:
Needs some explanation what this actually does. That it adds them to the `Cargo.toml` and not to the `#[cfg]`

Reply:
Good idea, I'll extend the explanations a bit.

Reply:
Maybe should also contain something about `Cargo.toml` in the name of the configuration

Reply:
I'm open to suggestions. :)

Reply:
`dox_feature_dependencies`? It's dependencies after all